package.json

```diff
@@ -73,5 +74,6 @@
         "vfile-reporter-position": "^0.1.7",
         "vfile-reporter-pretty": "^6.1.0"
     },
+    "internalDependencies": "/unified|bail|is-plain-obj|trough|vfile.*|unist.*|hast.*|property-information|html-void-elements|.*-separated-tokens|.*entities.*|ccount|rehype*|string-width|strip-ansi|ansi-regex|supports-color|rehype|web-namespaces|zwitch/",
```

Comment:
This is no longer needed, now that we have the function, right?

Reply:
Right, I forgot that